config/locales/en.yml

```diff
@@ -155,6 +155,10 @@ en:
     date: Date
     related_content_title: Related content
     not_applicable: Not applicable
+    dispute: This asset is a subject matter of dispute
```

Comment:
I think this should say "This asset is the subject matter of dispute" or "This asset is subject matter of dispute". The inclusion of the 'a' scans funny to me. And in the guidance it is referred to using either 'the' or no article at all